tmt/steps/provision/podman.py

```diff
@@ -315,7 +315,9 @@ def go(self) -> None:
         # Prepare data for the guest instance
         data_from_options = {
             key: self.get(key)
-            for key in PodmanGuestData.keys()
+            # SIM118: Use `{key} in {dict}` instead of `{key} in {dict}.keys()`.
+            # "Type[PodmanGuestData]" has no attribute "__iter__" (not iterable)
+            for key in PodmanGuestData.keys()  # noqa: SIM118
```

Comment:
Correct, `PodmanGuestData` (and various `*Data` classes) are not dictionaries. They do have `keys/values/items` methods, but are not iterable. Would it help if they become more dict-like & gain `__iter__`?

Reply:
Probably not for me to answer...or to say whether such change should be part of this PR for that matter :)

Reply:
Filed https://github.com/teemtee/tmt/issues/2049